### DISCUSSION TITLE
Testnets not reachable on Android client

### DISCUSSION THREAD

**Discussion by c03rad0r** [markqvist/Reticulum#449] [Help & Questions]
I can see traffic & announcements on testnet via TCP in the desktop client, but not on the Android client. 

How can I debug the issue? 

Is there a way to edit the config file manually in Android?

**Comment by markqvist** [comment]
_Please read the [Contribution Guidelines](https://github.com/markqvist/Reticulum/blob/master/Contributing.md). Specifically, using the Issue Tracker for posting support requests and general questions decreases visibility of your question for other users that could help you, and means that core development time is used on it instead._

_This question will be moved to the appropriate discussion section._

**Comment by markqvist** [comment]
Simply enabling "Connect via TCP" in the **Connectivity** section, with the default address and port (`sideband.connect.reticulum.network` port `7822`), and then restarting Sideband (use "Shutdown" in the menu, then relaunch) will enable a connection to the public testnet via TCP.

This works as expected on Android, I verified it just now on a couple of devices. If you are not getting this behaviour, it is more likely to be an IP/DNS connectivity issue on your network. You could try testing DNS lookup, TCP connectivity and such from a computer on the same network, or from within a terminal on the Android device.
